app/models/cmr.rb

```diff
@@ -412,6 +414,8 @@ def self.random_granules_from_collection(collection_concept_id, granule_count =
       granule_data_list = []
 
       #getting a random list of granules addresses within available amount
+      # CMR limits the page depth to 1,000,000, see ticket CMRARC-480
+      total_granules = [1000000, total_granules].min
```

Comment:
does it include the max number?

Reply:
... excludes max
https://ruby-doc.org/core-2.1.3/Range.html